base/setup/reactos/resource.h

```diff
@@ -59,19 +61,21 @@
 #define IDD_RESTARTPAGE      2070
 #define IDC_FINISHTITLE      2071
 #define IDC_RESTART_PROGRESS 2072
-#define IDC_PARTMOREOPTS     2073
 
 #define IDD_BOOTOPTIONS      2080
 #define IDC_PATH             2081
 #define IDC_INSTFREELDR      2082
 #define IDC_INSTVBRONLY      2083
 #define IDC_NOINSTFREELDR    2084
 
-#define IDD_PARTITION        2090
-#define IDC_UPDOWN1          2091
-#define IDC_PARTSIZE         2092
-#define IDC_UNIT             2093
-#define IDC_FSTYPE           2094
+#define IDD_PARTITION           2090
+#define IDC_EDIT_PARTSIZE       2091
+#define IDC_UPDOWN_PARTSIZE     2092
+#define IDC_UNIT                2093
+#define IDC_CHECK_MBREXTPART    2094
+#define IDC_FS_STATIC           2095
+#define IDC_FSTYPE              2096
+#define IDC_CHECK_QUICKFMT      2097
```

Comment:
superfluous and counterproductive indentation change
```suggestion
#define IDD_PARTITION        2090
#define IDC_EDIT_PARTSIZE    2091
#define IDC_UPDOWN_PARTSIZE  2092
#define IDC_UNIT             2093
#define IDC_CHECK_MBREXTPART 2094
#define IDC_FS_STATIC        2095
#define IDC_FSTYPE           2096
#define IDC_CHECK_QUICKFMT   2097
```